apis/v1beta1/vspherevm_types.go

```diff
@@ -32,6 +34,11 @@ const (
 	// IPClaimFinalizer allows the reconciler to prevent deletion of an
 	// IPAddressClaim that is in use.
 	IPAddressClaimFinalizer = "vspherevm.infrastructure.cluster.x-k8s.io/ip-claim-protection"
+
+	// GuestShutdownDefaultTimeout is the default timeout to wait for
+	// guest shutdown finishes before powering off the VM forcibly
+	// Only effective when the powerOffMode is set to trySoft.
+	GuestShutdownDefaultTimeout = 60 * time.Minute
```

Comment:
@srm09 , are we ok with a different timeout to VM Operator?

Reply:
I believe the timeout to VM Operator is set to 5 minutes.
Let me know if we should keep the same with VM Operator until the timeout can be set in VM Operator.

Reply:
I think we should keep them in sync.

Reply:
Sure. Reverted it back to 5 minutes.